### DISCUSSION TITLE
官网同款的编辑器的语法提示有开源吗？

### DISCUSSION THREAD

**Discussion by tanjiancheng** [baidu/amis#10159] [Show and tell]
官网同款的编辑器的语法提示有开源吗？感觉这个官网可以开放一套出来跟着版本走

**Comment by 2betop** [comment]
什么功能？

**Reply by tanjiancheng** [reply]
![image](https://github.com/baidu/amis/assets/7401570/e36e247f-f68a-43e2-afe8-cef93bc7a3db)
类似图示的中文提示和语法提示功能

**Comment by 2betop** [comment]
这个功能需要以下两步开启

1. 先把 amis npm 里面的 schema.json 部署到服务器上，比如可以通过 http://127.0.0.1/schema.json
2. 编辑器 component 配置 $schemaUrl 属性为 `http://127.0.0.1/schema.json`

**Reply by tanjiancheng** [reply]
再次请问下第一步的schema.json 要怎么样生成？

**Reply by 2betop** [reply]
npm run build-schemas --workspace=amis

**Reply by tanjiancheng** [reply]
![image](https://github.com/baidu/amis/assets/7401570/8c854046-83c6-4912-beef-4400e20dd8d1)


好像没有build-schemas这个命令哦

**Reply by 2betop** [reply]
https://github.com/baidu/amis/blob/master/packages/amis/package.json#L16
